swank/backend.lisp

```diff
@@ -719,7 +719,7 @@ return the results and T.  Otherwise, return the original form and
 NIL."
   (let ((fun (and (consp form)
                   (valid-function-name-p (car form))
-                  (compiler-macro-function (car form)))))
+                  (compiler-macro-function (car form) env))))
```

Comment:
How does this fix work? swank-macrostep is not passing an environment to `compiler-macroexpand-1` is it?